pkg/proto/nbcontract/v1/kubeletconfig.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.33.0
+// 	protoc-gen-go v1.35.1
```

Comment:
Should there be a matching version change in a go.mod file or somewhere like that?

Reply:
The makefile downloads the binary `buf` every time to compile to .go files. So it doesn't have go.mod file.